app/konnect/mesh-manager/migrate-zone.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Migrate a self-managed global zone control plane to {{site.konnect_saas}}
+content_type: tutorial
+---
+
+This guide explains how to migrate a self-managed global control plane to {{site.konnect_saas}}. This can be useful if you manage other services in {{site.konnect_saas}} that way you can view all your entities in one place. 
```

Comment:
Can we get something a bit more selling? It should probably link back to the overview to explain what mesh manager is